bindings/node/etc/build-static.sh

```diff
@@ -11,22 +11,68 @@ mkdir -p $BUILD_DIR
 mkdir -p $BUILD_DIR/bson-build
 mkdir -p $BUILD_DIR/libmongocrypt-build
 
-pushd $BUILD_DIR
+# Copied from the mongo-c-driver
```

Comment:
this seems specific to evergreen, can we leave it in a utilities file there which you `source` in here, or better yet run it before this file is run and ensure that `cmake` is in the `PATH` (perhaps have it installed to the `$(pwd)/bin` mentioned above)

Reply:
I'll try it out